page-template-cluster.php

```diff
@@ -80,6 +88,9 @@
 			</div>
 		</div>
 	</div>
+	<!-- Promo Section One -->
+	<?php echo ! empty( $section_one ) ? do_shortcode( "[ucf-section id=\"$section_one->ID\"]" ) : ''; ?>
```

Comment:
Each section shortcode printed in this template will need a `title` param passed to it (to ensure an `aria-label` attr gets set on the generated `<section>`), in case the provided sections don't have an inner heading anywhere in their content